kvm/v12.3.0/release.diff

```diff
@@ -0,0 +1,51 @@
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v12.2.0                                                 |    name: v12.3.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                         |      - name: cert-exporter
```

Comment:
Could you fix the indentation of `release.yaml` for v12.3.0 so we don't have a diff showing every line changed?

Reply:
Good catch, i'm on it